openlibrary/catalog/add_book/__init__.py

```diff
@@ -640,7 +642,7 @@ def load_data(rec, account=None):
     return reply
 
 
-def load(rec, account=None):
+def load(rec, account_key: str = None):
```

Comment:
Mypy doesn't understand this syntax.
```suggestion
def load(rec, account_key=None):
```